students_generator.py

```diff
@@ -0,0 +1,62 @@
+import random
+
+
+def generate_student_name():
+    first_names = ["John", "Jane", "Emily", "Chris", "Alex", "Michael", "Emma", "Olivia",
+                   "James", "Isabella", "Sophia", "Amelia", "Lucas", "Charlotte", "Aiden","Mia",
+                   "Ethan", "Harper", "Mason", "Ella", "Ava", "William", "Sophie", "Benjamin", "Zoe"
+                   "Liam", "Lily", "Noah", "Grace", "Oliver", "Chloe", "Elijah", "Aria", "Logan",
+                   "Avery", "Alexander", "Evelyn", "Jacob", "Abigail", "Daniel", "Nora", "Matthew",
+                   "Scarlett", "David", "Madison", "Joseph", "Layla", "Jackson", "Riley", "Samuel"]
+
+    last_names = ["Doe", "Smith", "Davis", "Brown", "Johnson", "Wilson", "Moore", "Taylor",

```

Comment:
This should be moved to consts and defined before all funcs (at the top of the file)